src/report/report-system/html-barchart.scm

```diff
@@ -58,7 +60,7 @@
 
 (define (gnc:make-html-barchart)
   (gnc:make-html-barchart-internal -1 -1 #f #f #f #f '() '() '() 
-				   #f #f #f '() #f #f #f #f #f #f))
+				   #f #f #f #t #t '() #f #f #f #f #f #f))
```

Comment:
It's the first two that parameters here that should change to '(relative . -1) to be initialized as a pair.